lib/rules/sotd/mailing-list.js

```diff
@@ -45,8 +45,8 @@ exports.check = function (sr, done) {
         if (!foundRepo) {
```

Comment:
That means changing the requirements for RECs. Can we do that without consulting spec-prod before?